Sources/HaCWebsiteLib/WorkshopManager.swift

```diff
@@ -1,62 +1,118 @@
 import Foundation
 import DotEnv
+import Dispatch
+import Yaml
 
 public struct WorkshopManager {
```

Comment:
Leaving a note here that Git has awkwardly turned deleting one file and renaming another into a merge of the two files. This is why there's this weird diff going on.